src/azure-cli/azure/cli/command_modules/appservice/static_sites.py

```diff
@@ -599,6 +599,68 @@ def get_user_function(cmd, name, resource_group_name):
     return client.get_user_provided_function_apps_for_static_site(name=name, resource_group_name=resource_group_name)
 
 
+def validate_backend(cmd, name, resource_group_name, backend_resource_id,
+                     backend_region=None, environment_name='default'):
+    from azure.mgmt.web.models import StaticSiteLinkedBackendARMResource
+
+    parsed_rid = parse_resource_id(backend_resource_id)
+    backend_name = parsed_rid["name"]
+
+    backend = StaticSiteLinkedBackendARMResource(
+        backend_resource_id=backend_resource_id,
+        region=backend_region)
+
+    client = _get_staticsites_client_factory(cmd.cli_ctx)
+    return client.begin_validate_backend_for_build(
+        name=name,
+        resource_group_name=resource_group_name,
+        linked_backend_name=backend_name,
+        static_site_linked_backend_envelope=backend,
+        environment_name=environment_name)
+
+
+def link_backend(cmd, name, resource_group_name, backend_resource_id, backend_region=None, environment_name='default'):
+    from azure.mgmt.web.models import StaticSiteLinkedBackendARMResource
+
+    parsed_rid = parse_resource_id(backend_resource_id)
+    backend_name = parsed_rid["name"]
+
+    backend = StaticSiteLinkedBackendARMResource(
+        backend_resource_id=backend_resource_id,
+        region=backend_region)
+
+    client = _get_staticsites_client_factory(cmd.cli_ctx)
+    return client.begin_link_backend_to_build(
+        name=name,
+        resource_group_name=resource_group_name,
+        linked_backend_name=backend_name,
+        static_site_linked_backend_envelope=backend,
+        environment_name=environment_name)
+
+
+def unlink_backend(cmd, name, resource_group_name, remove_backend_auth=False, environment_name='default'):
+    if remove_backend_auth:
+        logger.warning("--remove-backend-auth: removing auth configuration from backend")
```

Comment:
Could we make the warning log clearer and more user-friendly?
```suggestion
        logger.warning("Please note that using parameter --remove-backend-auth will remove auth configuration from backend")
```